modules/electromagnetics/test/tests/kernels/coupled_electrostatics/tests

```diff
@@ -0,0 +1,72 @@
+[Tests]
+  [coupled_electrostatics_3d_hex27]
+    type = 'Exodiff'
+    input = 'coupled_electrostatics.i'
+    exodiff = 'coupled_electrostatics_3d_hex27_out.e'
+    cli_args = 'Outputs/file_base=coupled_electrostatics_3d_hex27_out'
+    requirement = 'The system shall be capable of modeling a div-grad problem '
+                  'in H(div) and in 3d using HEX27 Raviart-Thomas elements.'
+    design = 'DivField.md GradField.md VectorDivPenaltyDirichletBC.md'
+    issues = '#26105'
```

Comment:
Since these use the same issue, `issues = '#26105'` could be moved to a higher level of the tree (just under `[Tests]`) to tidy up a few more lines.